docs/src/_includes/components/rule.macro.html

```diff
@@ -31,6 +31,8 @@
         <span class="visually-hidden">Categories:</span>
         {%- if params.deprecated -%}
             <p class="rule__categories__type">❌</p>
+            {%- elif params.categories and params.categories.frozen %}
+                <p class="rule__categories__type"> ❄️ <span class="visually-hidden">Frozen</span> </p>
```

Comment:
`no-extra-boolean-cast` is both recommended and frozen, but after this change, the recommended symbol won't be shown any more on the overview:

![image](https://github.com/user-attachments/assets/d280f820-bde3-4ae3-8954-9792dd0e2d8f)

Maybe it would be better to show both symbols, especially since we may decide to freeze more rules later?

Reply:
The layout is really only designed to support three icons. So I'd say if something is both recommended and frozen then we should favor showing the recommended checkmark. We can still show the frozen icon on the rule page.

Reply:
<img width="990" alt="image" src="https://github.com/user-attachments/assets/ef0a971f-2611-42fa-a4d2-181c519e2d05">

Would it look good to display the frozen icon next to the rule name?

Reply:
> Would it look good to display the frozen icon next to the rule name?

Works for me.

Reply:
Ooh yes, good idea @amareshsm :+1:

Reply:
sure I will make the changes accordingly.